src/client/components/localisation/view.tsx

```diff
@@ -3,19 +3,22 @@ import { runInAction } from 'mobx'
 import { observer } from 'mobx-react'
 import * as React from 'react'
 import { WebGLRenderer } from 'three'
+import { lazyInject } from '../../../inversify.config'
```

Comment:
Coming back to this in another PR! Name suggestions welcome, but `inversify.config` is the standard/recommended.